lambda-notes/src/Components/NoteList.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const NoteList = () => {
+  return(
+    <div>
+      NoteList
```

Comment:
If you're feeling stuck on the API calls, you can always start by working with dummy data and/or pseudocode what you'll be doing once the data is pulled in. For example, you probably know you'll need a map here, so you can note that if it helps you with forward momentum.